### DISCUSSION TITLE
Failed to launch because the 'phpstorm' scheme does not have a registered handler.

### DISCUSSION THREAD

**Discussion by philg124** [spatie/ignition#137] [Q&A]
Hi,

I have a fresh copy of windows 11 and PHPStorm 2022.1. When the ignition error page is shown any attempt to click the lines to open the file in PHPStorm fail.

In the Chrome console the following error is reported:

`Failed to launch 'phpstorm://open?file=XXX&line=52' because the scheme does not have a registered handler`

This is also the same in Edge.

Is there a step I am missing? Do I need any specific plugin for this to work?

**Comment by Abdelraman** [comment]
having the same issue
